### ISSUE TITLE
Unity3D Recipes are failing.

### ISSUE THREAD

**Issue by paul-cossey** [autopkg/joshua-d-miller-recipes#130]
Hi, @joshua-d-miller 

The Unity3D recipes are currently failing.

```
autopkg run -v /Users/paul/Documents/GitHub/AutoPkg\ Repos/joshua-d-miller-recipes/Unity3D/Unity3D.munki.recipe 
Processing /Users/paul/Documents/GitHub/AutoPkg Repos/joshua-d-miller-recipes/Unity3D/Unity3D.munki.recipe...
WARNING: /Users/paul/Documents/GitHub/AutoPkg Repos/joshua-d-miller-recipes/Unity3D/Unity3D.munki.recipe is missing trust info and FAIL_RECIPES_WITHOUT_TRUST_INFO is not set. Proceeding...
URLTextSearcher
No match found on URL: https://unity3d.com/unity-release/latest
Failed.
Receipt written to /Users/paul/Library/AutoPkg/Cache/com.github.joshua-d-miller.autopkg.munki.Unity3D/receipts/Unity3D.munki-receipt-20221207-165405.plist

The following recipes failed:
    /Users/paul/Documents/GitHub/AutoPkg Repos/joshua-d-miller-recipes/Unity3D/Unity3D.munki.recipe
        Error in com.github.joshua-d-miller.autopkg.munki.Unity3D: Processor: URLTextSearcher: Error: No match found on URL: https://unity3d.com/unity-release/latest

Nothing downloaded, packaged or imported.
```

https://unity3d.com/unity-release/latest give a 404 error when browsing to it

https://unity3d.com website forward's to https://unity.com 

This seems to be the latest release page https://unity.com/releases/editor/whats-new/2022.1.24 

It's the end of the day here, I'll try and knock up a PR tomorrow if you don't get round to it.

Thanks!

**Comment by paul-cossey** [comment]
Hi, @joshua-d-miller 

This commit https://github.com/autopkg/joshua-d-miller-recipes/pull/128/commits/9d829eefda67dd1d33ce07f8a5b33e31e88e070c fixes the unity issue. 

I've also updated the Munki recipe so that it pulls the correct min_os_version of 10.14 rather than a default of 10.5.0

Output from a verbose run

```
autopkg run -v /Users/paul/Documents/GitHub/AutoPkg\ Repos/joshua-d-miller-recipes/Unity3D/Unity3D.munki.recipe
Processing /Users/paul/Documents/GitHub/AutoPkg Repos/joshua-d-miller-recipes/Unity3D/Unity3D.munki.recipe...
WARNING: /Users/paul/Documents/GitHub/AutoPkg Repos/joshua-d-miller-recipes/Unity3D/Unity3D.munki.recipe is missing trust info and FAIL_RECIPES_WITHOUT_TRUST_INFO is not set. Proceeding...
URLTextSearcher
URLTextSearcher: Found matching text (match): https://download.unity3d.com/download_unity/35dcd44975df/MacEditorInstallerArm64/Unity-2022.2.0f1.pkg
URLDownloader
URLDownloader: Storing new Last-Modified header: Wed, 07 Dec 2022 14:37:01 GMT
URLDownloader: Storing new ETag header: "82794d6c2a4c0014dd6eadd5ddc8067a"
URLDownloader: Downloaded /Users/paul/Library/AutoPkg/Cache/com.github.joshua-d-miller.autopkg.munki.Unity3D/downloads/Unity 3D.pkg
CodeSignatureVerifier
CodeSignatureVerifier: Verifying installer package signature...
CodeSignatureVerifier: Package "Unity 3D.pkg":
CodeSignatureVerifier:    Status: signed by a developer certificate issued by Apple for distribution
CodeSignatureVerifier:    Signed with a trusted timestamp on: 2022-12-05 18:25:58 +0000
CodeSignatureVerifier:    Certificate Chain:
CodeSignatureVerifier:     1. Developer ID Installer: Unity Technologies ApS (BVPN9UFA9B)
CodeSignatureVerifier:        Expires: 2027-02-01 22:12:15 +0000
CodeSignatureVerifier:        SHA256 Fingerprint:
CodeSignatureVerifier:            00 05 6B 23 B9 38 0A A1 53 C7 4E 3C 1F EF 8A 3D D8 7C 18 B9 91 B6 
CodeSignatureVerifier:            8E D5 65 E4 62 21 09 D1 80 38
CodeSignatureVerifier:        ------------------------------------------------------------------------
CodeSignatureVerifier:     2. Developer ID Certification Authority
CodeSignatureVerifier:        Expires: 2027-02-01 22:12:15 +0000
CodeSignatureVerifier:        SHA256 Fingerprint:
CodeSignatureVerifier:            7A FC 9D 01 A6 2F 03 A2 DE 96 37 93 6D 4A FE 68 09 0D 2D E1 8D 03 
CodeSignatureVerifier:            F2 9C 88 CF B0 B1 BA 63 58 7F
CodeSignatureVerifier:        ------------------------------------------------------------------------
CodeSignatureVerifier:     3. Apple Root CA
CodeSignatureVerifier:        Expires: 2035-02-09 21:40:36 +0000
CodeSignatureVerifier:        SHA256 Fingerprint:
CodeSignatureVerifier:            B0 B1 73 0E CB C7 FF 45 05 14 2C 49 F1 29 5E 6E DA 6B CA ED 7E 2C 
CodeSignatureVerifier:            68 C5 BE 91 B5 A1 10 01 F0 24
CodeSignatureVerifier: 
CodeSignatureVerifier: Signature is valid
CodeSignatureVerifier: Authority name chain is valid
EndOfCheckPhase
FlatPkgUnpacker
FlatPkgUnpacker: Unpacked /Users/paul/Library/AutoPkg/Cache/com.github.joshua-d-miller.autopkg.munki.Unity3D/Downloads/Unity 3D.pkg to /Users/paul/Library/AutoPkg/Cache/com.github.joshua-d-miller.autopkg.munki.Unity3D/Unpack/
PkgRootCreator
PkgRootCreator: Created /Users/paul/Library/AutoPkg/Cache/com.github.joshua-d-miller.autopkg.munki.Unity3D/application_payload/Applications
PkgPayloadUnpacker
PkgPayloadUnpacker: Unpacked /Users/paul/Library/AutoPkg/Cache/com.github.joshua-d-miller.autopkg.munki.Unity3D/Unpack/Unity.pkg.tmp/Payload to /Users/paul/Library/AutoPkg/Cache/com.github.joshua-d-miller.autopkg.munki.Unity3D/application_payload/Applications
MunkiInstallsItemsCreator
MunkiInstallsItemsCreator: Created installs item for /Applications/Unity/Unity.app
MunkiInstallsItemsCreator: Derived minimum os version as: 10.14.0
MunkiPkginfoMerger
MunkiPkginfoMerger: Merged {'installs': [{'CFBundleIdentifier': 'com.unity3d.UnityEditor5.x', 'CFBundleName': 'Unity', 'CFBundleShortVersionString': 'Unity version 2022.2.0f1', 'CFBundleVersion': '2022.2.0f1', 'minosversion': '10.14.0', 'path': '/Applications/Unity/Unity.app', 'type': 'application', 'version_comparison_key': 'CFBundleVersion'}], 'minimum_os_version': '10.14.0'} into pkginfo
Versioner
Versioner: Found version 2022.2.0f1 in file /Users/paul/Library/AutoPkg/Cache/com.github.joshua-d-miller.autopkg.munki.Unity3D/application_payload/Applications/Unity/Unity.app/Contents/Info.plist
MunkiPkginfoMerger
MunkiPkginfoMerger: Merged {'version': '2022.2.0f1'} into pkginfo
MunkiImporter
MunkiImporter: Using repo lib: AutoPkgLib
MunkiImporter:         plugin: FileRepo
MunkiImporter:           repo: /Users/Shared/munki_repo
MunkiImporter: Copied pkginfo to: /Users/Shared/munki_repo/pkgsinfo/apps/unity/Unity 3D-2022.2.0f1__3.plist
MunkiImporter:            pkg to: /Users/Shared/munki_repo/pkgs/apps/unity/Unity 3D-2022.2.0f1__3.pkg
Receipt written to /Users/paul/Library/AutoPkg/Cache/com.github.joshua-d-miller.autopkg.munki.Unity3D/receipts/Unity3D.munki-receipt-20221213-150144.plist

The following new items were downloaded:
    Download Path                                                                                              
    -------------                                                                                              
    /Users/paul/Library/AutoPkg/Cache/com.github.joshua-d-miller.autopkg.munki.Unity3D/downloads/Unity 3D.pkg  

The following new items were imported into Munki:
    Name      Version     Catalogs  Pkginfo Path                             Pkg Repo Path                          Icon Repo Path  
    ----      -------     --------  ------------                             -------------                          --------------  
    Unity 3D  2022.2.0f1  testing   apps/unity/Unity 3D-2022.2.0f1__3.plist  apps/unity/Unity 3D-2022.2.0f1__3.pkg
```

**Comment by joshua-d-miller** [comment]
This has been committed. This can now be closed.
